_episodes_rmd/06-best-practices-R.Rmd

```diff
@@ -3,11 +3,13 @@ title: "Best Practices for Writing R"
 teaching: 10
 exercises: 0
 questions:
-- "How can I write R that other people can understand and use?"
+- "How can I write R code that other people can understand and use?"
```

Comment:
How about this, compared to https://github.com/swcarpentry/r-novice-inflammation/pull/383#discussion_r203262857?

Reply:
I agree. But maybe we could also change the title to "Best Practices for Writing R Code"? :)

Unfortunately, I don't have the time right now to push changes, but please feel free to go ahead with this.